src/Barryvdh/TranslationManager/Manager.php

```diff
@@ -137,7 +137,7 @@ public function exportTranslations($group)
             foreach($tree as $locale => $groups){
                 if(isset($groups[$group])){
                     $translations = $groups[$group];
-                    $path = $this->app->make('path').'/lang/'.$locale.'/'.$group.'.php';
+                    $path = $this->app->make('path').'/lang.corr/'.$locale.'/'.$group.'.php';
```

Comment:
I did ask to ignore that little chunk. I was not planning to have it in the pull request but missed it when I made the commit.

To explain why lang.corr. In my case as I am working now, I generate the translation files for all changes coming from translators, using some cobbled together scripts and csv import/export, into lang.corr (language corrections) and then merge them with a diff tool. That way I get to review the changes and don’t lose the comments in the files.

I don’t trust the translations to be overwritten. Sometimes the context is not clear and the corrected translation is wrong.

I’ll try to be more careful with future pull requests. I’m still wobbly with git. I made a separate branch that I will use for changes destined for pull requests. I plan on making lots of customizations specific to the project I am using it on so not all changes will be applicable to the package. I am hoping to catch the extra changes locally by having to merge into a local branch first.

BTW, thank you again for a great job.

Best regards,


Vladimir.

> On Mar 19, 2015, at 9:10 AM, Barry vd. Heuvel <notifications@github.com> wrote:
> 
> In src/Barryvdh/TranslationManager/Manager.php <https://github.com/barryvdh/laravel-translation-manager/pull/35#discussion_r26751466>:
> 
> > @@ -137,7 +137,7 @@ public function exportTranslations($group)
> >              foreach($tree as $locale => $groups){
> >                  if(isset($groups[$group])){
> >                      $translations = $groups[$group];
> > -                    $path = $this->app->make('path').'/lang/'.$locale.'/'.$group.'.php';
> > +                    $path = $this->app->make('path').'/lang.corr/'.$locale.'/'.$group.'.php';
> lang.corr?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/barryvdh/laravel-translation-manager/pull/35/files#r26751466>.
>